src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -154,9 +161,39 @@ public InMoov(String n) {
   // ---------------------------------------------------------------
 
   public void attach(Attachable attachable) {
+    //opencv
     if (attachable instanceof OpenCV) {
       opencv = (OpenCV) attachable;
+      subscribe(opencv.getName(), "publishClassification");
+      //SpeechRecognizer
+    } else if (attachable instanceof SpeechRecognizer) {
+      if (attachable.getClass().getSimpleName().equals("WebkitSpeechRecognition")) {
+        WebGui webgui = (WebGui) Runtime.create("webgui", "WebGui");
+        webgui.autoStartBrowser(false);
+        webgui.startService();
+        webgui.startBrowser("http://localhost:8888/#/service/" + attachable.getName());
```

Comment:
you should not force the browser to start here.  This method is for attaching.  Please remove this code.